client/components/marketing-survey/cancel-purchase-form/step-components/upsell-step.tsx

```diff
@@ -116,6 +116,7 @@ export default function UpsellStep( { upsell, site, purchase, ...props }: StepPr
 	const couponCode = 'BIZC25';
 	const builtByURL =
 		'https://builtbywp.com/get-started/?utm_medium=automattic_referred&utm_source=WordPresscom&utm_campaign=cancel-flow';
+	const refundAmount = { props };
```

Comment:
@dzver I think this is meant to be `const { refundAmount } = props;` instead :)

Reply:
Also, along with this, many occurrences of `props.refundAmount` can be just `refundAmount` below :)